src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    @include square($half-size);
+    margin-right: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url("./../images/star.svg");
+  }
+
+  .stars--1 &__star:nth-child(-n+1),
+  .stars--2 &__star:nth-child(-n+2),
+  .stars--3 &__star:nth-child(-n+3),
+  .stars--4 &__star:nth-child(-n+4),
+  .stars--5 &__star:nth-child(-n+5) {
+    background-image: url("./../images/star-active.svg");
+  }
```

Comment:
12. [SASS] - Try using for loops for the stars selector.

Reply:
done)